Jenkinsfile

```diff
@@ -120,7 +120,7 @@ node('docker && imgtec') {  // Only run on internal slaves as build takes a lot
              + 'CONFIG_PACKAGE_kmod-usb-net-kaweth=y\n' \
              + 'CONFIG_PACKAGE_kmod-usb-net-mcs7830=y\n' \
              + 'CONFIG_PACKAGE_kmod-usb-net-pegasus=y\n' \
-             + 'CONFIG_PACKAGE_kmod-usb-net-qmi-wwa=y\n' \
+             + 'CONFIG_PACKAGE_kmod-usb-net-qmi-wwan=y\n' \
```

Comment:
For now it's part of OpkgRemove test, I'd leave this in until we've cleaned all of those kmods out to leave only the essential ones.